ssl/ssl_ciph.c

```diff
@@ -101,10 +101,7 @@ static const ssl_cipher_table ssl_cipher_table_cipher[SSL_ENC_NUM_IDX] = {
     {SSL_CHACHA20POLY1305, NID_chacha20_poly1305},
 };
 
-static const EVP_CIPHER *ssl_cipher_methods[SSL_ENC_NUM_IDX] = {
-    NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
-    NULL, NULL
-};
+static const EVP_CIPHER *ssl_cipher_methods[SSL_ENC_NUM_IDX];
```

Comment:
I made this change based on a comment by Rich back in January :)  I've removed it now.  I didn't realise there were repercussions on some platforms -- BTW the initialisation list is too short for the array so currently there is a mixture of explicit and implicit initialisers.

I'd not noticed the different segments for the two declarations.  That's interesting.  A little weird but gcc often can be.